index.html

```diff
@@ -2905,6 +2915,168 @@ <h3>Evidence</h3>
 
       </section>
 
+      <section>
+        <h3>Confirmation Method</h3>
+
+        <p>
+A Confirmation Method can be included by an <a>issuer</a> in a <a>verifiable
+credential</a> to declare that the <a>subject</a> controls, or a <a>holder</a>
+has been designated to use, one or more particular confirmation methods and to
+bind the <a>claims</a> about the <a>subject</a> to one or more of these
+confirmation methods. In this way, an <a>issuer</a> explicitly gives guidance
+and enables a <a>verifier</a> to validate that the <a>holder</a> presenting the
+<a>verifiable credential</a> has proven control of one or more of these
+confirmation methods when the <a>claims</a> bound to the confirmation method
+are presented.
+        </p>
+
+        <p class="note">
+A <a>verifier</a> can decide to accept <a>claims</a> in a <a>verifiable
+credential</a> without validating the confirmation method, or to use a different
+mechanism to validate whether the <a>holder</a> is bound to the presented <a>claims</a>,
+if required. Such decision can prevent impact on the <a>verifier</a>'s
+liability or lack thereof if not specified by other means such
+as a <code>termsOfUse</code> policy.
+        </p>
+
+        <p>
+This specification defines the <code>confirmationMethod</code> <a>property</a>
+for expressing confirmation method information in a
+<code>credentialSubject</code> in a <a>verifiable credential</a>.
+        </p>
+
+        <div class="note">
+          <p>
+For example, an <a>issuer</a> can include a confirmation method based on public
+key cryptography in the <a>verifiable credential</a>. A <a>holder</a> can
+generate and include a <a>proof</a> with a cryptographic signature in the
+<a>verifiable presentation</a>, where the verification key of the cryptographic
+signature is bound to a confirmation method in the embedded <a>verifiable
+credential</a>.
+          </p>
+          <p>
+A <a>verifier</a> can validate that the <a>holder</a> controls
```

Comment:
what about substituting *can* with *may*?

Reply:
Is the issuer's intention that a verifier "SHOULD" ? why else include a holder binding?

Reply:
Note, that this is informative text. VCDM 2.0 is still a data model spec. That is why the text says "can".

Reply:
I think people are nudging you towards "MAY" and "SHOULD" as the two options due to this old chestnut:
https://www.rfc-editor.org/rfc/rfc2119

I think CAN makes perfect sense for an extension, but might be distracting for readers raised on the RFC2119 vocab who are expecting MAY for the same meaning.  SHOULD (or more specifically, SHOULD NOT) might be useful when describing how verifiers process confirmation methods :D

Reply:
The VCDM SHOULD only use normative verbs if we can write tests in our test suite. That was my point.

Reply:
> [any spec] SHOULD only use normative verbs if we can write tests in our test suite. That was my point.

YES! And optimally, testing SHOULD be possible and implemented for ALL normatively verbed features, including MAY and SHOULD and all the negatives (MAY NOT, SHOULD NOT, MUST NOT), etc., as well as MUST.

Reply:
No update in one moth, I guess we can resolve this.